src/base/ApplicationECSAlbCodeDeploy.ts

```diff
@@ -71,7 +72,8 @@ export class ApplicationECSAlbCodeDeploy extends Construct {
           },
           terminateBlueInstancesOnDeploymentSuccess: {
             action: 'TERMINATE',
-            terminationWaitTimeInMinutes: 5,
+            terminationWaitTimeInMinutes:
+              (this.config.successTerminationWaitTimeInMinutes ??= 5),
```

Comment:
ah didnt know we can do `??=`...usually have seen `A ?? B` to choose B if A is undefined

Reply:
If you think ?? is best I can update - here is documentation I used for that: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment

Reply:
no need to update..its cool! TIL `??=`

Reply:
may be we can default to 5 if someone set it to zero, 
```
(this.config.successTerminationWaitTimeInMinutes > 0) ?  (this.config.successTerminationWaitTimeInMinutes ) : 5
```

Reply:
had a slack discussion with Nate, the docs allow 0 as an option, no need to implement this constraint